docs/source/guides/io.ipynb

```diff
@@ -346,12 +346,64 @@
    "source": [
     "### Text based file formats\n",
     "\n",
-    "Datasets are still commonly stored in text-based file formats such as CSV. Since text-based file formats are not memory-mappable, they have to be read in memory. If the contents of a CSV file fits into the available RAM, one can simply do:"
+    "Datasets are still commonly stored in text-based file formats such as CSV and JSON. Vaex supports various methods for reading such datasets.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\"><b>New in 4.14: </b> \n",
+    "\n",
+    "The `vaex.open` method can also be used to read a CSV file. With this method Vaex will *lazily* read the CSV file, i.e. the data from the CSV file will be streamed when computations need to be executed. This is powered by Apache Arrow under the hood. In this way you can work with arbitraruly large CSV files without caring about RAM!\n",
+    "\n",
+    "Note: When opening a CSV file in this way, Vaex will first quickly scan the file to determine some basic metadata such as the number of rows, column names and their data types. The duration of this can vary depending on the number of rows, columns, your disk read speed, and settings telling Vaex thoroughly it should scan the CSV file. \n",
```

Comment:
Should we at mention the `schema_infer_fraction` here?

Reply:
Made some changes, now it is explicitly mentioned. What do you think?